indico/modules/designer/models/templates.py

```diff
@@ -64,9 +65,8 @@ class DesignerTemplate(db.Model):
         JSON,
         nullable=False
     )
-    background_image_id = db.Column(
```

Comment:
I wouldn't even go as far as discussing the pros of using a proper relationship. The previous solution was perfectly fine, and the changes we agreed don't change anything about it.